buf.yaml

```diff
@@ -10,6 +10,7 @@ lint:
   except:
     - SERVICE_SUFFIX
     - RPC_REQUEST_STANDARD_NAME
+    - RPC_REQUEST_RESPONSE_UNIQUE
```

Comment:
added this line here to prevent the linter from complaining about reusing response types. imo doesn't make sense to have unique response types for rpc that return the exact same structure

Reply:
let's not do this sorry... https://docs.buf.build/lint/rules#rpc_request_standard_name-rpc_response_standard_name-rpc_request_response_unique